homeassistant/components/home_connect/strings.json

```diff
@@ -461,6 +461,7 @@
         "laundry_care_washer_enum_type_spin_speed_r_p_m400": "400 rpm",
         "laundry_care_washer_enum_type_spin_speed_r_p_m600": "600 rpm",
         "laundry_care_washer_enum_type_spin_speed_r_p_m800": "800 rpm",
+        "laundry_care_washer_enum_type_spin_speed_r_p_m900": "900 rpm",
```

Comment:
Side note: The format of the key is weird. I'd expect it to be `laundry_care_washer_enum_type_spin_speed_rpm_900`